.changeset/flat-clouds-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-i18n": patch
```

Comment:
I notice that only i18n is flagged as having changes, even though the build-script will almost certainly change all the packages (and require republishing them all).

Reply:
Good catch.  I'll move the build-scripts into build-settings.